pkg/ctl/delete/cluster.go

```diff
@@ -108,19 +108,18 @@ func doDeleteCluster(cmd *cmdutils.Cmd) error {
 	{
 
 		logger.Info("cleaning up LoadBalancer services")
```

Comment:
i'd move that log message inside the block

Reply:
Done.